### PR TITLE
Load from deeplink

### PR DESCRIPTION
**Pull request by Erikvv**
Intial version to load pilot from a URL search parameter. This can use some UX improvement but the change was getting pretty big.

### CODE REVIEW

File: webtool/frontend/src/components/main-content.tsx [hunk 1]
```diff
@@ -18,13 +19,6 @@ export const MainContent: FunctionComponent<EmotionProps> = () => {
         setShowSimulation(false)
     }
 
-    useEffect(() => {
-        // Set Default HouseholdGroup, minimum required to get run the simulation
-        const defaultHouseholdGroup = new HouseholdGroup("Huishoudens", 200, 0.2, 0.1, 0.2, 0.0, 3000);
```
Comment (macano) [inline webtool/frontend/src/components/main-content.tsx:23]: I think after this fix, the simulation was fixed to run without setting up the minimum required. Did you test that the simulation won't be an error if no household is defined?

File: webtool/gradle/common/src/commonMain/kotlin/Pilot.kt [hunk 2]
```diff
@@ -68,6 +68,27 @@ data class Pilot(
...
 
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+fun pilotFromJson(json: String) = Json.decodeFromString<Pilot>(json)
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+fun createDefaultStartPilot() = Pilot(
+    name = "start",
+    householdGroups = listOf(
```
Comment (macano) [inline webtool/gradle/common/src/commonMain/kotlin/Pilot.kt:79]: I think it works without an initial setup, it's been fixed and the simulation side. Check with @GillisHommen

**Review by macano — APPROVED** [review]
LGTM

File: webtool/gradle/common/src/commonMain/kotlin/Pilot.kt (hunk 2 above)
Comment (Erikvv) [reply webtool/gradle/common/src/commonMain/kotlin/Pilot.kt:79]: It works without households. @GillisHommen do you want the initial households removed?

File: webtool/frontend/src/components/main-content.tsx (hunk 1 above)
Comment (Erikvv) [reply webtool/frontend/src/components/main-content.tsx:23]: The web app still initializes a household group, but the simultation works without.

**Pull request merged**